spring-cloud-sleuth-core/src/main/java/org/springframework/cloud/sleuth/Span.java

```diff
@@ -397,7 +425,30 @@ public long getSpanId() {
 	}
 
 	/**
-	 * A pseudo-unique (random) number assigned to the trace associated with this span
+	 * When non-zero, the trace containing this span uses 128-bit trace identifiers.
+	 *
+	 * <p>{@code traceIdHigh} corresponds to the high bits in big-endian format and
+	 * {@link #getTraceId()} corresponds to the low bits.
+	 *
+	 * <p>Ex. to convert the two fields to a 128bit opaque id array, you'd use code like below.
+	 * <pre>{@code
+	 * ByteBuffer traceId128 = ByteBuffer.allocate(16);
+	 * traceId128.putLong(span.getTraceIdHigh());
+	 * traceId128.putLong(span.getTraceId());
+	 * traceBytes = traceId128.array();
+	 * }</pre>
+	 *
+	 * @see #traceIdString()
+	 * @since 1.2.0
```

Comment:
I think it is worthwhile to backport this to 1.0 and 1.1, even though it would be annoying. The reason is that these versions will be around a long time, and truncate trace ids perpetually unless we do..